lib/teleterm/api/proto/v1/access_request.proto

```diff
@@ -0,0 +1,48 @@
+// Copyright 2021 Gravitational, Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.terminal.v1;
+
+option go_package = "github.com/gravitational/teleport/lib/teleterm/v1";
+
+message AccessRequest {
+  string id = 1;
+  string state = 2;
+  string resolve_reason = 3;
+  string request_reason = 4;
+  string user = 5;
+  repeated string roles = 6;
+  string created = 7;
+  string expires = 8;
+  repeated AccessRequestReview reviews = 9;
+  repeated string suggested_reviewers = 10;
+  repeated string threshold_names = 11;
+  repeated ResourceID resource_ids = 12;
+}
+
+message AccessRequestReview {
+  string author = 1;
+  repeated string roles = 2;
+  string state = 3;
+  string reason = 4;
+  string created = 5;
+}
+
+message ResourceID {
+  string kind = 1;
+  string name = 2;
+  string cluster_name = 3;
+}
```

Comment:
Do we use this structure for compatibility with access request stuff from webapps.e? If not then teleterm could probably use resource URIs instead but I understand if that's not viable here.

Reply:
This is for webapps.e compatibility, yeah